app/templates/bower.json

```diff
@@ -6,22 +6,29 @@
     "angular-bootstrap": "~0.13.3",
     "angular-cookies": "~1.4.4",
     "angular-highlightjs": "~0.4.3",
+    "angular-mocks": "~1.4.4",
     "angular-ui-router": "~0.2.15",
     "angular-ui-tinymce": "~0.0.9",
+    "angular-x2js": "https://github.com/janmichaelyu/angular-x2js.git",
```

Comment:
Just wondering, no dependency of x2js, or is that fully included in angular-x2js?